server/graphql/v2/query/collection/OrdersCollectionQuery.ts

```diff
@@ -197,13 +200,18 @@ export const OrdersCollectionResolver = async (args, req: express.Request) => {
       'data.fromAccountInfo.name',
       'data.fromAccountInfo.email',
     ],
+    emailFields: isHostAdmin ? ['$createdByUser.email$'] : [],
     amountFields: ['totalAmount'],
     stringArrayFields: ['tags'],
     stringArrayTransformFn: (str: string) => str.toLowerCase(), // expense tags are stored lowercase
   });
 
   if (searchTermConditions.length) {
     where[Op.and].push({ [Op.or]: searchTermConditions });
+    include.push({
+      association: 'createdByUser',
+      attributes: [],
+    });
```

Comment:
To save a `JOIN` when we're not fetching the email

```suggestion
    if (searchTermConditions.emailFields.includes('$createdByUser.email$')) {
      include.push({
        association: 'createdByUser',
        attributes: [],
      });
    }
```